app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "paho",
```

Comment:
I would name both the file and the id as  `mqtt` to reflet the connector name, paho

Reply:
Done, please review again. Thanks, Luca.